docs/disaster-recovery.md

```diff
@@ -0,0 +1,443 @@
+# Failover and Failback In DR
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/)
+ is an asynchronous replication of RBD images between multiple Ceph clusters.
+ This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded
+ to the associated journal before modifying the actual image.
+ The remote cluster will read from this associated journal and
+ replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or
+ manually created RBD image mirror-snapshots to replicate
+ crash-consistent RBD images between clusters.
+
+Here, `snapshot based mirroring` approach is used to asynchronously
+replicate RBD images for Disaster Recovery.
+
+## Create Ceph Pools
+
+In this section, we create specific ceph pools that are RBD mirroring
+ enabled for use with the DR use case.
+
+>:bulb: **Note:** It is also feasible to edit existing pools and enable
+> them for replication.
+
+For information on how to deploy a ceph pool and enable mirroring on it,
+ please refer to the
+ [ceph documentation](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#pool-configuration).
+
+>:memo: WARNING: Pool name across the cluster peers should be the
+> same for RBD replication to function.
+
+## Bootstrap Peers
```

Comment:
Lets not have different links for below items

* Create Pool
* Bootstrap Peers
* Create RBDMirror

Have a single link to configure rbd mirroring and provide a link to set up rbd mirroring.